crates/server/src/config/mod.rs

```diff
@@ -204,6 +207,14 @@ impl Config {
         }
     }
 
+    /// the HTTP endpoint from where requests are received
+    #[cfg(any(feature = "dns-over-https-rustls", feature = "dns-over-h3"))]
+    pub fn get_http_endpoint(&self) -> &str {
```

Comment:
Should avoid `get_()` prefixes per the API naming guidelines.

Reply:
In this case it's to maintain consistency with other `Config` functions like `get_tls_cert()` and `get_directory()`

Reply:
Okay, I guess.